typeclasses/characters.py

```diff
@@ -9,10 +9,11 @@
 """
 # import importlib
 from evennia import DefaultCharacter
+from evennia.utils import lazy_property
 from utils.trait import Trait
```

Comment:
For world I've been following what the game_template says, that it is for 'things which are not commands or typeclasses'. So I think ./world is OK for equip.py. 'World' perhaps needs a better name.